app/models/casa_case.rb

```diff
@@ -65,6 +69,15 @@ def update_cleaning_contact_types(args)
     end
   end
 
+  def clear_court_dates
```

Comment:
I think the scope should be used in here, or else all court dates might get wiped... not just the ones in scope?